the first task.cpp

```diff
@@ -1,7 +1,9 @@
-#include <iostream>
+#include <iostream> // 2/2
 
-int Main()
+int main() // 0/1
 {
-	std::cout << "Hello, World!\n";
-		return 0;
-}
\ No newline at end of file
+	std::cout << "Hello, World!\n"; // 4/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line